invenio_jsonschemas/views.py

```diff
@@ -46,9 +48,17 @@ def create_blueprint(state):
     def get_schema(schema_path):
         """Retrieve a schema."""
         try:
-            return send_from_directory(state.get_schema_dir(schema_path),
-                                       schema_path)
+            schema_dir = state.get_schema_dir(schema_path)
         except JSONSchemaNotFound:
             abort(404)
+        if request.args.get(
+            'refs',
+            current_app.config.get('JSONSCHEMAS_REPLACE_REFS'),
+            type=int
+        ):
+            schema = {'$ref': request.base_url}
```

Comment:
Yep, the first day, I tried with different parameters including `base_uri` and the only way I managed to make it work was starting from `{'$ref': 'http://localhost:5000/schemas/records/hep.json'}` and letting the resolver do the job of resolving all $ref recursively.